dev-util/idea-ultimate/idea-ultimate-15.0.0.143.249.6_pre.ebuild

```diff
@@ -0,0 +1,91 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+inherit eutils versionator
+
+SLOT="0"
+PV_STRING="$(get_version_component_range 4-6)"
+MY_PV="$(get_version_component_range 1-3)"
+MY_PN="idea"
+
+DESCRIPTION="A complete toolset for web, mobile and enterprise development"
+HOMEPAGE="http://www.jetbrains.com/idea"
+SRC_URI="https://download.jetbrains.com/idea/${MY_PN}IU-${PV_STRING}.tar.gz"
+
+LICENSE="IDEA 
+    || (IDEA_Academic IDEA_Classroom IDEA_OpenSource IDEA_Personal)"
+IUSE=""
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="!dev-util/${PN}:14
+	!dev-util/${PN}:15"
+RDEPEND="${DEPEND}
+	>=virtual/jdk-1.7"
```

Comment:
`virtual/jdk` is slotted, so you have to decide if it will keep working once you replace e.g. jdk-1.7 with jdk-1.8. If it will work without rebuilding, you want `:*` here. If it needs rebuilding on java upgrade, you need `:=`.